### PR TITLE
Mani/dev 5233/list item component

### PR DESCRIPTION
**Pull request by maniart**
- Add List View and List Item 
screen shot 2018-01-03 at 3 14 53 pm

- Add `Pro Badge` example to `docs.md` for easier use.

### CODE REVIEW

**Comment by parsakamali** [comment]
@mani can you lighten the separator to match the one below the list? I think ADK has a couple border/separator classes…

**Comment by dennymcfadden** [comment]
@maniart great docs on the pro badge! do we have a text style that would be slightly smaller and bolder? Like this: https://www.dropbox.com/s/1ndqr25d3wm7x35/Screenshot%202018-01-03%2015.28.16.png?dl=0

File: scss/_adk-type.scss [hunk 1]
```diff
@@ -1,3 +1,10 @@
+// ADK Font Weights
+@each $weight, $value in $adk-font-weights {
```
Comment (maniart) [inline scss/_adk-type.scss]: @parsakamali @dennymcfadden Added utility classes for font-weight following the example from `fs-*` - I looked into the repo and couldn't locate any existing font-weights, so I added these. If they exist and I missed them, let me know.

File: scss/_adk-variables.scss [hunk 2]
```diff
@@ -76,6 +76,13 @@ $adk-font-sizes: (
   l: $space-l,
 );
 
+$adk-font-weights: (
```
Comment (maniart) [inline scss/_adk-variables.scss]: here's all the font-weights, according to `_adk-fonts.scss`

File: docs.md [hunk 3]
```diff
@@ -1016,7 +1016,7 @@ For the sake of convenience, here is the markup and css classes required to rend
   </div>
   <!--  Pro Badge -->
   <div class="columns shrink pl-0">
-    <span class="blue capitalize">PRO</span>
+    <span class="blue capitalize fs-s fw-bold">PRO</span>
```
Comment (maniart) [inline docs.md]: `fw-*` usecase

**Comment by maniart** [comment]
@dennymcfadden updated Pro badge style
screen shot 2018-01-03 at 3 52 48 pm

**Comment by parsakamali** [comment]
I don't think we had font-weight utilities, this all looks great!

**Pull request merged**